src/System.Diagnostics.EventLog/src/System/Diagnostics/EventLog.cs

```diff
@@ -265,7 +280,7 @@ public static void CreateEventSource(EventSourceCreationData sourceData)
                 {
                     Debug.WriteLineIf(CompModSwitches.EventLog.TraceVerbose, "CreateEventSource: SourceExists returned true");
                     if (".".Equals(machineName))
-                        throw new ArgumentException(SR.Format(SR.LocalSourceAlreadyExists, source));
+                        throw new ArgumentException(SR.Format(SR.LocalSourceAlreadyExists, source, machineName));
```

Comment:
this isn't right, the old code had these strings:
LocalSourceAlreadyExists=Source {0} already exists on the local computer.
SourceAlreadyExists=Source {0} already exists on the computer '{1}'.

the localsourcealreadyexists doesn't take a machine name. Your resx is wrong.